projects/1.20/assets/simply-tea/simplytea/lang/zh_cn.json

```
@@ -0,0 +1,58 @@
+{
+  "itemGroup.simplytea": "简茶",
+
+  "block.simplytea.tea_sapling": "茶树树苗",
+  "block.simplytea.tea_trunk": "茶树树干",
+  "block.simplytea.tea_fence": "茶枝栅栏",
+  "block.simplytea.tea_fence_gate": "茶枝栅栏门",
+
+  "item.simplytea.tea_leaf": "茶叶",
+  "item.simplytea.tea_leaf.tooltip": "令人愉悦的清香",
+  "item.simplytea.black_tea": "红茶",
+  "item.simplytea.black_tea.tooltip": "「酵」",
+  "item.simplytea.tea_stick": "茶枝",
+  "item.simplytea.tea_stick.tooltip": "散发着肉桂般的香气",
+  "item.simplytea.chorus_petal": "紫颂花瓣",
+  "item.simplytea.chorus_petal.tooltip": "神秘之花",
+  "item.simplytea.ice_cube": "冰块",
+  "item.simplytea.ice_cube.tooltip": "冰霜凝结",
+
+  "item.simplytea.unfired_cup": "未烧制的茶杯",
+  "item.simplytea.cup": "茶杯",
+  "item.simplytea.cup_water": "清水",
+  "item.simplytea.cup_tea_green": "绿茶",
+  "item.simplytea.cup_tea_black": "红茶",
+  "item.simplytea.cup_tea_floral": "蒲公英花茶",
+  "item.simplytea.cup_tea_chai": "印度奶茶",
+  "item.simplytea.cup_tea_iced": "苹果冰茶",
+  "item.simplytea.cup_tea_chamomile": "黄春菊花茶",
+  "item.simplytea.cup_tea_chorus": "紫颂花茶",
+  "item.simplytea.cup_cocoa": "热可可",
+  "item.simplytea.cup.with_honey": "调有蜂蜜",
+  "item.simplytea.cup.with_cinnamon": "调有肉桂",
+
+  "item.simplytea.teabag": "茶包",
+  "item.simplytea.teabag_green": "绿茶包",
+  "item.simplytea.teabag_black": "红茶包",
+  "item.simplytea.teabag_floral": "蒲公英花茶包",
+  "item.simplytea.teabag_chamomile": "黄春菊花茶包",
+  "item.simplytea.teabag_chorus": "紫颂花茶包",
+
+  "item.simplytea.unfired_teapot": "未烧制的茶壶",
+  "item.simplytea.teapot": "茶壶",
+  "item.simplytea.teapot.tooltip": "空",
+  "item.simplytea.teapot_water": "茶壶",
+  "item.simplytea.teapot_water.tooltip": "盛满：水",
+  "item.simplytea.teapot_milk": "茶壶",
+  "item.simplytea.teapot_milk.tooltip": "盛满：牛奶",
+  "item.simplytea.teapot_hot": "茶壶",
+  "item.simplytea.teapot_hot.tooltip": "小心烫伤！",
+  "item.simplytea.teapot_frothed": "茶壶",
+  "item.simplytea.teapot_frothed.tooltip": "表层漂浮着温热的奶泡",
```

原文：Frothed milk!
[询问]这茶壶是装满了奶泡还是只是“浮有”
[考量]如下
```suggestion
  "item.simplytea.teapot_frothed.tooltip": "浮有奶泡",
```

---

其实是装满了加热后的奶

---

那可以考虑仿照前面“满满一壶x”这种写法
比如“满壶热奶泡”....吗？

---

```suggestion
  "item.simplytea.teapot_frothed.tooltip": "满壶热奶泡",
```
好耶（）

---

```suggestion
  "item.simplytea.teapot_frothed.tooltip": "满壶热奶泡",
```
好耶（）